probes/http/http.go

```diff
@@ -411,9 +411,20 @@ func (p *Probe) startForTarget(ctx context.Context, target endpoint.Endpoint, da
 	result := p.newResult()
 	req := p.httpRequestForTarget(target, nil)
 
+	firstRun := p.c.GetSkipFirstInterval()
+
 	ticker := time.NewTicker(p.opts.Interval)
 	defer ticker.Stop()
-	for ts := range ticker.C {
```

Comment:
Instead of adding a config option, can you please just change this for loop to:

for ts := time.Now() ; true; ts =<- ticker.C {
  ...
}

This will have desired effect.

Reply:
Updated